Core/include/Acts/TrackFitting/detail/VoidKalmanComponents.hpp

```diff
@@ -84,4 +84,21 @@ struct VoidOutlierFinder {
   }
 };
 
+/// @brief void smoothing logic
+struct VoidReverseFilteringLogic {
+  /// @brief Public call mimicking an outlier finder
+  ///
+  /// @tparam track_state_t Type of the track state
+  ///
+  /// @param trackState The trackState to investigate
```

Comment:
Can you specify the @param reversedFiltering bool 

- and should the Void component not only return false ?

Reply:
Done